File: reddio-csharp/reddio.api/V1/Rest/Contract.cs
using Newtonsoft.Json;

namespace Reddio.Api.V1.Rest
{
    public class Contract
    {
        [JsonProperty("contract_address")] public string ContractAddress;
        [JsonProperty("symbol")] public string Symbol;
        [JsonProperty("decimals")] public long Decimals;
        [JsonProperty("type")] public string Type;
        [JsonProperty("quantum")] public string Quantum;
        [JsonProperty("asset_type")] public string AssetType;

        public Contract(string contractAddress, string symbol, long decimals, string type, string quantum, string assetType)
        {
            ContractAddress = contractAddress;
            Symbol = symbol;
            Decimals = decimals;
            Type = type;
            Quantum = quantum;
            AssetType = assetType;
        }

        public Contract()
        {
        }
    }
}

File: reddio-java/reddio-api/src/main/java/com/reddio/api/v1/rest/Contract.java
package com.reddio.api.v1.rest;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor(staticName = "of")
public class Contract {
    @JsonProperty("contract_address")
    private String contractAddress;
    @JsonProperty("symbol")
    private String symbol;
    @JsonProperty("decimals")
    private Long decimals;
    @JsonProperty("type")
    private String type;
    @JsonProperty("quantum")
    private Long quantum;
    @JsonProperty("asset_type")
    private String assetType;
}
